src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/SchoolYearValidation.feature

```diff
@@ -0,0 +1,208 @@
+@SchoolYear
+Feature: School Year Reference Validation
+  Reference validation on School Years
+
+  """
+  @StudentEducationOrganizationAssociation Hook that will create the following data for the required tests
+  createStudent(studentUniqueId)
+  createSchool(schoolId)
+  createCourse(courseCode, schoolId)
+  createSession(schoolId, schoolYear, sessionName)
+  createCourseOffering(localCourseCode, courseCode, schoolId, schoolYear, sessionName)
+  createSection(localCourseCode, schoolId, schoolYear, sessionName, sectionIdentifier)
+  """
```

Comment:
What do you think about adding a `Background` section that includes `Given 2024 is a valid school year`, for example? Or something similar to that.

Reply:
Understood. I was considering both options—using `Hooks or Background`—but I wasn't certain if the elements defined in the Background section are necessary for all scenarios in the Feature file, if that's the case I think we can use a `background` section

Reply:
I'm not as familiar with hooks. Does the hook allow you to see the data here in the Feature file?

For example, I would want the reader to see what years are available just by looking at the Feature file, without having to look in the backend code.